flexmeasures/data/services/scheduling.py

```diff
@@ -201,10 +207,8 @@ def load_custom_scheduler(scheduler_specs: dict) -> Tuple[Callable, str]:
     assert "module" in scheduler_specs, "scheduler specs have no 'module'."
     assert "function" in scheduler_specs, "scheduler specs have no 'function'"
 
-    source_name = scheduler_specs.get(
-        "source", f"custom scheduler - {scheduler_specs['function']}"
-    )
     scheduler_name = scheduler_specs["function"]
+    source_info = dict(model=scheduler_name, version="1", name="")  # default  # default
```

Comment:
An empty DataSource name as a default would not be my preference. Maybe "unknown author" would be better suited to prompt developers to set the `__author__` variable.

(The inline comment is repeated.)